cmd/commands/server.go

```diff
@@ -6,12 +6,19 @@ import (
 )
 
 func NewServerCommand() *cobra.Command {
+	var (
+		port   int
+		secure bool
```

Comment:
How about use `insecure` and default to `false`? This is more common.

Reply:
will do, yeah that seems more natural.  saw argo-workflows had `secure` so initially had gone with that approach.